### DISCUSSION TITLE
CARLA 0.9.14 Cannot load new map

### DISCUSSION THREAD

**Discussion by Kin-Zhang** [carla-simulator/carla#6066] [Q&A] [answered]
Downloaded the release version (tar) and extracted it.

After additional maps imported, 0.9.14 still cannot load the map (Other maps can be loaded correctly)

Success on Town10HD:
![image](https://user-images.githubusercontent.com/35365764/209563692-89e0852d-945f-4360-b4af-0d5693f6cae1.png)

But failed on new map feature in 0.9.14:
![image](https://user-images.githubusercontent.com/35365764/209563745-e0136978-123e-4d31-8c22-7cd2884018e0.png)

![image](https://user-images.githubusercontent.com/35365764/209563768-5693f287-c957-4725-a6a1-6c331cc0dd99.png)

---

Tested the release and committed version: 0.9.14 https://github.com/carla-simulator/carla/commit/580ae28c9a33f6c0307fd1737d37d2856360fda0

@bernatx

**Comment by bernatx** [comment, marked as answer]
Hi,
The problem is that Town11 and Town12 are **large maps** built from separated tiles that makes a huge map. With large maps, when you load the map you don't see anything until you add a **hero** vehicle (using the script manual_control.py for example). Then the map loads the tile where the hero vehicle is placed, and when it moves through the map, the appropriate tiles are loaded and unloaded in realtime. That is why you don't see anything yet. That makes possible to have huge maps loaded, because only the tiles around the hero vehicle are loaded.

Regards,
Bernat

**Reply by Kin-Zhang** [reply]
THANKS!!
